setup.bzl

```diff
@@ -25,9 +25,11 @@ def stardoc_repositories():
     _include_if_not_defined(
         http_archive,
         name = "bazel_skylib",
-        urls = ["https://github.com/bazelbuild/bazel-skylib/archive/16de038c484145363340eeaf0e97a0c9889a931b.tar.gz"],  # 2020-08-11
-        sha256 = "96e0cd3f731f0caef9e9919aa119ecc6dace36b149c2f47e40aa50587790402b",
-        strip_prefix = "bazel-skylib-16de038c484145363340eeaf0e97a0c9889a931b",
+        sha256 = "58f558d04a936cade1d4744d12661317e51f6a21e3dd7c50b96dc14f3fa3b87d",
```

Comment:
note, this update is needed to print a better failure message on diff_test (there hasn't been a release there for over a year :(

Reply:
https://github.com/bazelbuild/bazel-skylib/releases/tag/1.1.1 released this week :)

Reply:
The test is, unfortunately, failing on Windows: https://buildkite.com/bazel/stardoc/builds/231#054cdb1e-52a4-45cc-8a62-463ccf788d76

Would you mind bumping the skylib dep to 1.1.1 (which has some additional Windows-specific fixes) to see if that allows the test to pass?

And if that doesn't help, we'd have to go back to the old test runner.

Reply:
Still fails, I should file a bug on skylib's diff_test I guess